getnacelle/nacelle-js
# fix: replace non-relay queries

### WHY are these changes introduced?

Fixes [[ENG-7224](https://nacelle.atlassian.net/browse/ENG-7224)] 



### WHAT is this pull request doing?



## 

- replace non-relay queries with relay queries

### How to Test

- `npm run generate && npm run start` (to test the route building with the new queries)
- make sure homepage works and all the content in it shows properly
- make sure the cart works
- navigate to a product page `product/handle`
- navigate to a collection page `collections/handle`
- navigate to a content page `content/handle`
